File: src/main/java/com/example/mapserialization/MapserializationApplication.java
package com.example.mapserialization;

import com.redis.om.spring.annotations.EnableRedisDocumentRepositories;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@EnableRedisDocumentRepositories
public class MapserializationApplication {

	public static void main(String[] args) {
		SpringApplication.run(MapserializationApplication.class, args);
	}

}

File: src/main/kotlin/com/example/mapserialization/MapserializationApplication.kt
package com.example.mapserialization

import com.redis.om.spring.annotations.EnableRedisDocumentRepositories
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
@EnableRedisDocumentRepositories
class MapserializationApplication

fun main(args: Array<String>) {
    runApplication<MapserializationApplication>(*args)
}
